.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint Python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit black codespell flake8 isort mypy pyupgrade safety
+      - run: bandit --recursive --skip B101,B108,B110,B310,B311,B403,B404,B603 .
```

Comment:
- I would add a comment on each of the linter used here, explain what they are testing.
- Open3D's Python package installation requires compiling C++/CUDA code. Could you point out which of these tests needs to be done after installation of the Open3D Python package?